### PR TITLE
updated templates

### PR DESCRIPTION
**Pull request by robch**

### CODE REVIEW

File: src/ai/.x/templates/openai-chat-streaming-with-functions-js/CreateOpenAI.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    return new OpenAI({
+      apiKey: azureOpenAIAPIKey,
+      baseURL: `${azureOpenAIEndpoint.replace(/\/+$/, '')}/openai/deployments/${azureOpenAIDeploymentName}`,
+      defaultQuery: { 'api-version': azureOpenAIAPIVersion },
+      defaultHeaders: { 'api-key': azureOpenAIAPIKey },
+    });
+  }
+}
+
+exports.CreateOpenAI = CreateOpenAI;
```
Comment (chschrae) [inline src/ai/.x/templates/openai-chat-streaming-with-functions-js/CreateOpenAI.js:34]: can't you just add 'export' in front of the class?

File: src/ai/.x/templates/openai-chat-streaming-with-functions-js/Main.js [hunk 2]
```diff
@@ -14,28 +19,52 @@ const rl = readline.createInterface({
...
+    process.stdout.write('\nAssistant: ');
+    await chat.getResponse(input, (content) => {
+      process.stdout.write(content);
     });
 
-    console.log(`\nAssistant: ${response}\n`);
+    process.stdout.write('\n\n');
   }
 
   console.log('Bye!');
```
Comment (chschrae) [inline src/ai/.x/templates/openai-chat-streaming-with-functions-js/Main.js:66]: Some of the prints were changes to process.stdout.write() should this also be changed?

File: src/ai/.x/templates/openai-chat-webpage-js/src/CreateOpenAI.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      apiKey: azureOpenAIAPIKey,
+      baseURL: `${azureOpenAIEndpoint.replace(/\/+$/, '')}/openai/deployments/${azureOpenAIDeploymentName}`,
+      defaultQuery: { 'api-version': azureOpenAIAPIVersion },
+      defaultHeaders: { 'api-key': azureOpenAIAPIKey },
+      dangerouslyAllowBrowser: true
+    });
+  }
+}
+
+exports.CreateOpenAI = CreateOpenAI;
```
Comment (chschrae) [inline src/ai/.x/templates/openai-chat-webpage-js/src/CreateOpenAI.js:37]: I think you can just put 'export' in front of the class definition

**Review by chschrae — APPROVED** [review]

File: src/ai/.x/templates/openai-chat-streaming-with-functions-js/Main.js (hunk 2 above)
Comment (robch) [reply src/ai/.x/templates/openai-chat-streaming-with-functions-js/Main.js:66]: not as used...

File: src/ai/.x/templates/openai-chat-streaming-with-functions-js/CreateOpenAI.js (hunk 1 above)
Comment (robch) [reply src/ai/.x/templates/openai-chat-streaming-with-functions-js/CreateOpenAI.js:34]: not as used...

File: src/ai/.x/templates/openai-chat-webpage-js/src/CreateOpenAI.js (hunk 3 above)
Comment (robch) [reply src/ai/.x/templates/openai-chat-webpage-js/src/CreateOpenAI.js:37]: I'll double check in next PR

**Pull request merged**